website/cue/reference/components/sinks/datadog_events.cue

```diff
@@ -46,11 +47,7 @@ components: sinks: datadog_events: {
 
 	support: sinks._datadog.support
 
-	configuration: {
-		default_api_key: sinks._datadog.configuration.default_api_key
-		endpoint:        sinks._datadog.configuration.endpoint
-		site:            sinks._datadog.configuration.site
-	}
```

Comment:
Remove the definitions of the shared datadog cue stuff when the other DD PRs are merged in (assuming this is the last one in).

Reply:
I will address this in the "mini refactor" PR